.gitignore

```diff
@@ -35,3 +35,4 @@ MultiTestDB.conf
 *.frozen.conf
 bioperl*.zip
 /modules/t/CLEAN.t
+.idea/
```

Comment:
Could you remove this from your commit? I know we have already got tonnes of IDE-specific (assuming Emacs counts as an IDE, it does to me) exclusions in this repo but we cannot possibly handle all of them, and more importantly this constitutes a separate logical change from fixing the handling of $HOME in _load_database_.

Reply:
Ok, I just hit the delete file from here. It seems to have added a delete action for the file in the PR. (still two files in PR - and .gitignore removed).